tests/components/tuya/__init__.py

```diff
@@ -428,9 +428,15 @@
         Platform.BINARY_SENSOR,
         Platform.SENSOR,
     ],
+    "ywcgq_h8lvyoahr6s6aybf": [
+        # https://github.com/home-assistant/core/issues/145932
+        Platform.NUMBER,
+        Platform.SENSOR,
+    ],
     "ywcgq_wtzwyhkev3b4ubns": [
-        # https://community.home-assistant.io/t/something-is-wrong-with-tuya-tank-level-sensors-with-the-new-official-integration/689321
-        # not (yet) supported
+        # https://github.com/home-assistant/core/issues/103818
```

Comment:
Same model, but refreshed to use diagnostic dump from an issue not from a community post